tests/utils/actions/click.ts

```diff
@@ -51,3 +51,7 @@ export async function switchShapeType(page: Page, shapeType: string) {
 export async function activeEmbed(page: Page) {
   await page.click('.resizable-img');
 }
+
+export async function focusCaption(page: Page) {
+  await page.click('.embed-editing-state>format-bar-button:nth-child(1)');
+}
```

Comment:
focus caption is not a common function, so should not include in the `utils/actions`